.github/codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 5%
```

Comment:
So every time if there is a drop then that could add up. Normally isn't this total coverage percentage?